src/Camera.py

```diff
@@ -93,6 +133,28 @@ def clean(self):
         self.camera.EndAcquisition()
         del self.camera
 
+
+class MockCamera:
+    n_cameras = 0
+
+    def __init__(self):
+        self._name = f"MockCamera{MockCamera.n_cameras}"
+        MockCamera.n_cameras += 1
+        self.data = np.random.randint(0, 255, size=(5, 4000, 3000), dtype='ubyte')
```

Comment:
i think we should actually agree on a format for RGB frames, and row-vs-col orderings, that both the camera abstraction and the screenwidget agree with.